corehq/motech/openmrs/tests/test_repeaters.py

```diff
@@ -122,7 +130,157 @@ def test_utf8_question(self):
         self.assertEqual(value, {u'/data/foo/b\u0105r': 'baz'})
 
 
-class DocTests(SimpleTestCase):
+joburg = SQLLocation(location_id='johannesburg')
+cape_town = SQLLocation(location_id='cape_town')
+FakeLocation = namedtuple('FakeLocation', 'location_id')
+
+
+does_not_exist_mock = mock.Mock(side_effect=SQLLocation.DoesNotExist)
```

Comment:
Just curious why you use mock heavily here, as opposed to setting up these users/locations (since it's anyway if `LocationHierarchyTestCase` is subclassed)

Reply:
Yeah, I was thinking about exactly this yesterday. I used mock to avoid needing the database for these tests. If I'd used the database they'd be simpler and more readable, but slower. ... But considering we've got loads of tests that use the database, are these tests going to make a noticeable difference to the time it takes to run all of them? Nope. ... :thinking: I'll add it to my to-do list to rewrite without the mocks.

Reply:
Oh, I asked because the tests were subclassing `TestCase` instead of `SimpleTestCase`